editor/code/src/client.ts

```diff
@@ -70,11 +74,12 @@ export function activate(context: ExtensionContext): void {
   }
   const stop = () => {
     if (client && client.isRunning()) {
-      client.stop();
+      client.dispose(2000).then(updateStatusBar); // EJGA: is this in milliseconds?
```

Comment:
It seems to be milliseconds but I only read about
```
      const timeout = setTimeout(() => panel.dispose(), 5000);
```
doesn't look like dispose takes an arg.

Reply:
I see this in my types file:
```
dispose(timeout?: number): Promise<void>;
```
similarly for `stop`

Reply:
`panel` doesn't take the argument, indeed. This is only for the node languageserver client.